libusb/os/windows_winusb.c

```diff
@@ -485,6 +485,19 @@ static int get_interface_by_endpoint(struct libusb_config_descriptor *conf_desc,
 	return LIBUSB_ERROR_NOT_FOUND;
 }
 
+static const struct libusb_interface_descriptor *get_interface_descriptor_by_number(struct libusb_device_handle *dev_handle, struct libusb_config_descriptor *conf_desc, int iface, uint8_t altsetting)
+{
+	int i;
```

Comment:
To be strict we always have an empty line after declarations.

Reply:
```suggestion
	int i;

```